tools/docker-compose/README.md

```diff
@@ -1,8 +1,50 @@
-docker build --no-cache=true --rm=true -t ansible/awx_devel:latest .
-docker run --name awx_test -it --memory="4g" --cpuset="0,1" -v /Users/meyers/ansible/:/awx_devel -p 8013:8013 -p 8080:8080 -p 27017:27017 -p 2222:22 ansible/awx_devel
+# Docker Compose for Dev container
+
+## How to start the Dev container
+
+In the root directory of your awx clone, run the following to build your docker image:
+
+```
+make docker-compose-build
```

Comment:
shouldn't you mention something about tagging the container? Or at least that it will take a million years to finish

Reply:
Haha, true.  I'll add something for that